docs/source/bentocloud/reference/deployment-details.rst

```diff
@@ -0,0 +1,181 @@
+   * - Time
+     - The timestamp of when each log entry is recorded.
+   * - Unique labels
+     - Labels that provide context for the logs, such as the node, cluster, and Deployment name.
+   * - Wrap lines
+     - Wrap long lines for easier readability.
+   * - Prettify JSON
+     - Formats JSON log entries to be more human-readable.
+   * - Deduplication
+     - Filters out duplicate log entries. Options including "None", "Exact", "Numbers" and "Signature" provide different methods to identify and remove repeated logs.
+   * - Display results
+     - Sort log entries chronologically, either from the newest or the oldest.
+   * - Download
+     - Download logs in text or JSON format.
+
+.. note::
+
+   The **Logging** tab integrates Grafana Loki. For more information, see the `Loki documentation <https://grafana.com/docs/loki/latest/>`_.
+
+Monitoring
+----------
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Property
+     - Description
+   * - Number of Replicas
+     - Displays the count of active replicas in the Deployment. It provides a detailed count for both the total number of replicas and the individual number of API Servers and Runners.
+   * - Request Percentile Response Times
+     - Displays the response time across different percentiles (for example, P95), giving insight into the range of response speeds that users may experience. It includes the total response time as well as the response time by API endpoints.
+   * - Request Volume
+     - Tracks the number of requests made to the Deployment over time, which is helpful in understanding the demand and traffic patterns. It includes the total request volume as well as the volume by API endpoints.
+   * - Request Success Rate (non-4|5xx responses)
+     - Indicates the proportion of requests that are successfully processed without any client-side (4xx) or server-side (5xx) errors. It includes the total success rate as well as the success rate by API endpoints.
+   * - CPU Usage
+     - Displays the CPU utilization of the Deployment, which includes the total usage as well as the usage by individual components.
+   * - Memory Usage
+     - Displays the amount of memory being used over time, which includes the total usage as well as the usage by individual components.
+   * - GPU Usage
+     - Displays the GPU utilization of the Deployment, which includes the total usage as well as the usage by individual components.
+   * - GPU Memory Usage
+     - Displays the memory usage on the GPU, offering insights into how memory-intensive the GPU tasks are. It includes the total usage as well as the usage by individual components.
+   * - GPU Memory Bandwidth Usage
+     - Displays the bandwidth usage of the GPU memory, providing data on how intensively the memory is being accessed and at what rate data is being transferred. It includes the total usage as well as the usage by individual components.
```

Comment:
@Haivilo Please help take a look at the Monitoring properties. Thanks!

Reply:
lgtm!